swat/cas/connection.py

```diff
@@ -419,66 +419,45 @@ def _get_server_features(self):
         return stype, version, out
 
     def _detect_protocol(self, hostname, port, protocol=None):
```

Comment:
Docstrings must be in the numpy docstring format.  This will need to be changed back.

Reply:
Done.